lms/djangoapps/course_home_api/outline/v1/views.py

```diff
@@ -177,6 +220,25 @@ def get(self, request, *args, **kwargs):
 
         return Response(serializer.data)
 
+    def _get_start_block(self, blocks, block):
+        if not blocks.get_children(block):
+            return block
+
+        children = blocks.get_children(block)
+        start_block = self._get_start_block(blocks, children[0])
+        return start_block
```

Comment:
Similar comment here (double get_children calls):

```
children = blocks.get_children(block)
return self._get_start_block(blocks, children[0]) if children else block
```

Though that one liner might be veering too much on the side of brevity, and an if/else might be clearer.

Reply:
Do you also need to get down to the lowest level? What if you just return the jump_to at the section level? Does that work? I think the courseware might be smart enough to still bring you the right spot.

Reply:
Yes, just tested this. We can just use jump_to at the section level! Good call.